roles/edpm_hosts_entries/tasks/main.yml

```diff
@@ -57,8 +57,8 @@
     insertbefore: BOF
     block: "{{ edpm_hosts_entries_block }}"
     marker: "# {mark}"
-    marker_begin: "START_HOST_ENTRIES_FOR_STACK: {{ edpm_stack_name | default(plan) }}"
-    marker_end: "END_HOST_ENTRIES_FOR_STACK: {{ edpm_stack_name | default(plan) }}"
+    marker_begin: "START_HOST_ENTRIES_FOR_STACK: {{ edpm_stack_name | default(edpm_hosts_entries_plan) }}"
+    marker_end: "END_HOST_ENTRIES_FOR_STACK: {{ edpm_stack_name | default(edpm_hosts_entries_plan) }}"
```

Comment:
I know you didn't cause this, but instead of defaulting the plan var, we need to just drop it. Plan isn't used with EDPM, and a default value of overcloud doesn't make sense and is a TripleO leftover.
I would then just change the marker message to be START_HOST_ENTRIES_FOR_EDPM, etc. It does not need to be dynamic based on any ansible var.